pkgs/tools/inputmethods/ibus-engines/ibus-bamboo/default.nix

```
@@ -44,6 +44,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/BambooEngine/ibus-bamboo";
     license = licenses.gpl3;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ astronaut0212 ];
+    maintainers = with maintainers; [ ];
```

```suggestion
    maintainers = [ ];
```
If possible, could you rewrite your commit to apply this change in order to ensure the formatting CI check passes?